bitbucket/pipelineService.js

```diff
@@ -0,0 +1,35 @@
+const createService = require('../utils').createService;
+const moment = require('moment');
+
+const url = (host) => {
+    return host + ['repositories', process.env.repo, 'pipelines/'].join('/');
```

Comment:
Is the any reason for appending the directory seperator at the end of pipelines? I prefer to have them not hardcoded in the url parts as it is much easier to join later on.

Reply:
Unfortunately, there is a reason. `https://api.bitbucket.org/2.0/repositories/{repoSlug}/pipelines/` works, `https://api.bitbucket.org/2.0/repositories/{repoSlug}/pipelines` does not and the API will return a 404. A little strange :/

Reply:
Of course, that's weird, all good then!